crates/sdk/src/wallet/store.rs

```diff
@@ -654,6 +655,13 @@ impl Store {
         ];
         self.pkhs.extend(pkhs);
 
+        self.addresses
+            .extend([(address_alias.clone(), validator_address.clone())]);
```

Comment:
```suggestion
        self.addresses
            .insert(address_alias.clone(), validator_address.clone());
```

Reply:
done in [03006d8](https://github.com/anoma/namada/pull/3093/commits/03006d891727ec1a624b55cdb8d1fa85ab227347)